spec/models/case_contact_spec.rb

```diff
@@ -82,20 +82,20 @@
     expect(case_contact.errors[:base]).to eq(["cannot edit case contacts created before the current quarter"])
   end
 
-  context "#update_cleaning_contact_types" do
+  describe "#update_cleaning_contact_types" do
```

Comment:
Same here... did you make this change because there's not a describe within context?

Reply:
context and describe do the same thing, it's just a matter of which one "sounds" better
In general, describe is for methods, and context is for data setup- but it is not a hard and fast rule https://www.betterspecs.org/#describe

Reply:
Yeah, that was my understanding. That's why I'm unsure as to why these were changed in this way.

Reply:
This usage of "describe" for a method is more correct than "context" for a method according to https://www.betterspecs.org/#describe above which I assume is the explanation :)